build-system/tasks/presubmit-checks.js

```diff
@@ -45,11 +42,67 @@ var forbiddenTerms = {
   'indexedDB': '',
   'openDatabase': '',
   'requestFileSystem': '',
-  '\\.scrollTop': '',
   'webkitRequestFileSystem': '',
   'debugger': '',
 };
 
+var bannedTermsHelpString = 'Please review viewport.js for a helper method or'
+  'mark with `/*OK*/` or `/*REVIEW*/` and consult the AMP team.';
+
+// Most of the forbidden property/method access banned on the
```

Comment:
done.

thought it might be too wordy, but it wasn't too bad when i tested it out on the command line.